io.openems.edge.timeofusetariff.awattar/src/io/openems/edge/timeofusetariff/awattar/TimeOfUseTariffAwattarImpl.java

```diff
@@ -115,16 +115,11 @@ protected void deactivate() {
 		this.channel(TimeOfUseTariffAwattar.ChannelId.HTTP_STATUS_CODE).setNextValue(httpStatusCode);
 
 		/*
-		 * Schedule next price update for 2 pm
+		 * Schedule next price update every hour
 		 */
 		var now = ZonedDateTime.now();
-		var nextRun = now.withHour(14).truncatedTo(ChronoUnit.HOURS);
-		if (now.isAfter(nextRun)) {
-			nextRun = nextRun.plusDays(1);
-		}
-
-		var duration = Duration.between(now, nextRun);
-		var delay = duration.getSeconds();
+		var nextRun = now.plusHours(1).truncatedTo(ChronoUnit.HOURS);
```

Comment:
Please add a comment why this is now needed every hour. From what I understood aWATTar does not anymore provide full day-ahead prices, but only 24 hours ahead, right?